index.html

```diff
@@ -35,95 +35,71 @@ <h3>Let's connect</h3>
             </ul>
     </div>
     </section>
-    <div class="worksec">
-        <article class="work">
-            <header class="work">
+    <section class="worksec">
+        <section class="work">
+            <div class="work">
                 <img class="work" src="Images/Snapshoot-Portfolio.png" alt="Avocode Snapshot"/>
-                <h1 class="work">Tonic</h1>
-            </header>
-            <ul class="worklist">
-                <li class="card-title">CANOPY</li>
-                <li class="counter"><img src="Images/Counter.png" alt="Counter"/></li>
-                <li class="work">Back End Dev</li>
-                <li class="counter"><img src="Images/Counter.png" alt="Counter"/></li>
-                <li class="work">2015</li>
-            </ul>
+                <h2 class="work">Tonic</h2>
+                <img class="workcanopy" src="Images/Canopy.png" alt="Canopy"/>
+            </div>          
             <p class="work">A daily selection of privately personalized reads; no accounts 
                 or sign-ups required.</p>
             <ul class="languages">
                 <li class="languages">html</li>
                 <li class="languages">css</li>
                 <li class="languages">javascript</li>
             </ul>
-            <p class="worksee">See Project</p>
-        </article>
+            <button class="worksee">See Project</button>
+        </section>
```

Comment:
- [ ] As the previous reviewer mentioned, you should stick to the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122). you can do so by using the same margins, font colors, font style, padding, buttons, etc.
- [ ]  Please add button interactions like hover, focus, etc to make your website functional for all buttons as the previous reviewer mentioned.

| Your design     | Figma design |
| :------------ |   :---:       | 
| ![image](https://user-images.githubusercontent.com/53402600/170608095-5b0b8548-a4f6-4477-8f22-fe1a8b207979.png)| ![image](https://user-images.githubusercontent.com/53402600/170607917-79d60bc0-3abd-44ef-ba3e-66749ef8d7ad.png) |